src/js/components/Form/stories/typescript/TypedForm.tsx

```diff
@@ -75,14 +86,27 @@ export const TypedForm = () => {
           />
           <FormField
             label="Size"
+            htmlFor="size"
+            id="size"
+            htmlFor="size"
+            id="size"
```

Comment:
Oops actually I just noticed we have `id="size"` twice here.
```suggestion
            id="size"
            htmlFor="size"
```